src/main/java/com/sproutsocial/nsq/Connection.java

```diff
@@ -273,6 +273,12 @@ else if (response != null) {
                 }
             }
         }
+        catch (EOFException e) {
```

Comment:
If the nsqd closes the connection while blocked on a read call, we shouldn't log an error at all.